### PR TITLE
Small changes + fix for dishing to people with changed display names

### PR DESCRIPTION
**Pull request by FrederikBolding**
Fixes an issue with dishing to people who have changed their display names.

Small changes based on feedback from #22 (@vojtechsimetka & @deamme )

Let me know if you have any issues or comments!

### CODE REVIEW

File: pointsbot.js [hunk 1]
```diff
@@ -31,12 +32,16 @@ function handleDish(event, room, client, auth) {
...
+    message = event.getContent().formatted_body;
+  }
   if (message.trim()[0] == ">") { //quoting another user, skip the quoted part
     matched = true;
     message = message.split("\n\n")[1];
   }
 
-  const re = /!\s*dish\s+(\S+)\s+(\S+)\s+points\s+to\s+(\S+)\s+for\s+([^\n]+)/gi;
+
+  const re = /!\s*dish\s+(\S+)\s+(\S+)\s+points\s+to\s+([^\n]+)\s+for\s+([^\n]+)/gi;
```
Comment (vojtechsimetka) [inline pointsbot.js:44]: I actually think that this already handles the condition on line 12, right? https://github.com/Giveth/giveth-bot/pull/24/files#diff-d656cec8a77f5db4f51c4a44749dc151R12

Comment (vojtechsimetka) [reply pointsbot.js:44]: The `\s` right after `!`

**Comment by krrisis** [comment]
No idea if there's still an answer needed on the Q by vojtech above but logically no objection from me :)

File: pointsbot.js (hunk 1 above)
Comment (FrederikBolding) [reply pointsbot.js:44]: Sorry, I'm not the most proficient in regex, so I can't explain why, but no it doesn't match the string which is why I added the other condition.

Comment (FrederikBolding) [reply pointsbot.js:44]: Oh, nevermind. Just read through it again. The condition is there to match all commands even with a space between "!" and the command. It's done before even executing the regex, and just before matching commands.

Comment (FrederikBolding) [reply pointsbot.js:44]: The "changed" message without the space isn't sent to the regex anyway, so it successfully matches on the unchanged message. But to register the command we still need to remove the space.

**Comment by FrederikBolding** [comment]
Should be ready to merge if @vojtechsimetka agrees with my comments on his comment 😄

File: pointsbot.js (hunk 1 above)
Comment (deamme) [reply pointsbot.js:44]: I agree with @FrederikBolding seems alright to me. I'll merged this

**Pull request merged**